File: test_scripts/python/client_test.py
from runagent import RunAgentClient

ra = RunAgentClient(
    agent_id="8f266444-72aa-4e6e-9173-0173b8ded54f",
    entrypoint_tag="minimal",
    local=True
    )


agent_results = ra.run(
    role="user",
    message="Analyze the benefits of remote work for software teams"
)

print(agent_results)

# ==================================

# from runagent import RunAgentClient

# ra = RunAgentClient(
#     agent_id="b80aface-9fc3-4db0-b4f8-d1a319215767",
#     entrypoint_tag="demo2_stream",
#     local=True
#     )

# for chunk in ra.run([
#         {"role": "user", "content": "Analyze the benefits of remote work "
#          "for software teams"}
# ]):
#     print(chunk)

File: test_scripts/go/client_test.go
package main

import (
	"context"
	"fmt"
	"log"
	"time"

	"github.com/runagent-dev/runagent-go/pkg/client"
)

func main() {
	agentClient, err := client.New(
		"acfef275-a0ff-4d80-89a6-e23490408e97",
		"minimal",
		true,
	)
	if err != nil {
		log.Fatalf("Failed to create client: %v", err)
	}
	defer agentClient.Close()

	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
	defer cancel()

	result, err := agentClient.Run(ctx, map[string]interface{}{
		"role":    "user",
		"message": "Analyze the benefits of remote work for software teams",
	})
	if err != nil {
		log.Fatalf("Failed to run agent: %v", err)
	}

	fmt.Printf("Result: %v\n", result)
}
